corehq/apps/smsforms/tests/test_app.py

```diff
@@ -255,3 +256,7 @@ def test_case_actions_are_removed_when_true(self):
         self.assertIsNone(xml.find('case').find('create'))
         self.assertIsNone(xml.find('case').find('update'))
         self.assertIsNone(xml.find('case').find('close'))
+
+    def test_raises_exception_if_xml_is_none(self):
+        with self.assertRaises(PartialSubmissionXMLIsNone):
+            _clean_xml_for_partial_submission(None, should_remove_case_actions=True)
```

Comment:
This test might not be necessary anymore. See other comment on this review.

Reply:
See fd6460155257de86f571919dd691afa81fd8637f